### ISSUE TITLE
Spoty file for Tile EU E45N021

### ISSUE THREAD

**Issue by felixcremer** [EarthyScience/RQADeforestation.jl#40]
There seems to be some corrupted file at tile E045N021T3.
SIG0_20210202T055815__VH_D110_E045N021T3_EU020M_V1M0R1_S1AIWGRDH.tif

This is the file and it fails to read that file with a GDAL offset error

**Comment by felixcremer** [comment]
Do we have some communication with EODC about these corrupted files?

**Comment by felixcremer** [comment]
I found another one. 
SIG0_20210302T052715__VH_D168_E045N015T3_EU020M_V1M0R1_S1AIWGRDH.tif
It seems as if there are only a few specific files effected, because there are other files in the tile that are ok. 
I tested with gdalinfo and it can also not open the file.

**Comment by felixcremer** [comment]
Answer from EODC Support:


In the beginning of the last year we performed a data version conversion of our available SIG0 data due to necessary changes in the filenaming as well as file metadata such as the scale factor. At the same time we kept the "old" versions for completeness reasons.

V01R01 -> V0M2R4, 
V1M0R1 -> V1M1R1.

With June 2023, we introduced another version V1M1R2, which we use for NRT processing.

Additionally, we did a gapfilling of missing and broken data, which has been identified during the conversion procedure. For this activity, we also used the V1M1R2 version. 

Long story short, we would like you to use V0M2R4, V1M1R1 and V1M1R2. They are consistent and no broken files should be there.

The files identified by you are indeed broken and were part of the gapfilling activity. Please find the files here:

/eodc/products/eodc.eu/S1_CSAR_IWGRDH/SIG0/V1M1R2/EQUI7_EU020M/E045N021T3/SIG0_20210202T055815__VH_D110_E045N021T3_EU020M_V1M1R2_S1AIWGRDH_TUWIEN.tif

/eodc/products/eodc.eu/S1_CSAR_IWGRDH/SIG0/V1M1R2/EQUI7_EU020M/E045N021T3/SIG0_20210202T055815__VV_D110_E045N021T3_EU020M_V1M1R2_S1AIWGRDH_TUWIEN.tif

**Comment by felixcremer** [comment]
For me this means, that we only look at the folders V0M2R4, V1M1R1 and V1M1R2 and then we should not run into invalid files.
